src/Scouting.elm

```diff
@@ -1,65 +1,121 @@
+        , view = view
+        , subscriptions = always Sub.none
+        }
+
 
 init : Model
-init = {scoutersName = "", scoutedTeamNumber = "", matchNumber = "", scoutedTeamDriverStationPosition = "", isPage = False}
-
-update : Msg -> Model -> (Model, Cmd Msg)
-update msg model = 
-    case msg of 
-        ScoutersNameInput string ->
-            ({model | scoutersName = string }, Cmd.none)
-        TeamNumberInput string ->
-            ({model | scoutedTeamNumber = string }, Cmd.none)
-        MatchNumberInput string ->
-            ({model | matchNumber = string }, Cmd.none)
-        DriverStationInput string ->
-            ({model | scoutedTeamDriverStationPosition = string }, Cmd.none)
+init =
+    { scoutersName = "", scoutedTeamNumber = Nothing, matchNumber = Nothing, scoutedTeamDriverStationPosition = "", isPressed = NotPressed }
+
+
+update : Msg -> Model -> ( Model, Cmd Msg )
+update msg model =
+    case msg of
+        ScoutersNameInput name ->
+            ( { model | scoutersName = name }, Cmd.none )
+
+        TeamNumberInput name ->
+            ( { model | scoutedTeamNumber = String.toInt name }, Cmd.none )
+
+        MatchNumberInput name ->
+            ( { model | matchNumber = String.toInt name }, Cmd.none )
+
+        DriverStationInput name ->
+            ( { model | scoutedTeamDriverStationPosition = name }, Cmd.none )
+
         Page state ->
-            ({model | isPage = state}, Cmd.none)
-
-view : Model ->  Html.Html Msg
-view model = 
-    if model.isPage == True then
-        Html.text model.scoutersName
-    else 
-
-    Html.pre [][ 
-        inputwriter "your name" ScoutersNameInput model.scoutersName
-        ,inputwriter "scouted team number" TeamNumberInput model.scoutedTeamNumber
-        ,inputwriter "match number" MatchNumberInput model.matchNumber
-        ,inputwriter  "driver station position" DriverStationInput model.scoutedTeamDriverStationPosition
-        , Html.button [onClick <| Page True] [Html.text "second page"] 
+            ( inputChecker model, Cmd.none )
+
+
+viewWhenPressed : Model -> Html.Html Msg
+viewWhenPressed model =
+    Html.pre []
+        [ inputWriter "your name" ScoutersNameInput model.scoutersName
+        , inputWriter "scouted team number" TeamNumberInput << unwrapToString <| model.scoutedTeamNumber
+        , inputWriter "match number" MatchNumberInput << unwrapToString <| model.matchNumber
+        , inputWriter "driver station position" DriverStationInput model.scoutedTeamDriverStationPosition
+        , Html.button [ onClick <| Page Pressed ] [ Html.text "second page" ]
         ]
+
+
+view : Model -> Html.Html Msg
+view model =
+    if model.isPressed == Pressed then
+        Html.pre []
+            [ Html.text model.scoutersName
+            , Html.text " your name\n"
+            , Html.text (unwrapToString <| model.scoutedTeamNumber)
+            , Html.text " scouted team number\n"
+            , Html.text (unwrapToString <| model.matchNumber)
```

Comment:
The pipes are redundant here.
However - you should use them to remove the parentheses, for instance:
```elm
, Html.text <| unwrapToString model.scoutedTeamNumber
```